ubipop/__init__.py

```diff
@@ -604,6 +604,18 @@ def _diff_lists_by_attr(self, list_1, list_2, attr):
 
         return diff
 
+    def _filter_pkgs_from_modules(self):
+        regex = r'\d+:'
```

Comment:
I'm a bit confused at the intent here.  I think it's to strip off the leading epoch from the RPMs, right?  Like:

  0:foo-1.2.3-4.el8.x86_64.rpm => foo-1.2.3-4.el8.x86_64.rpm

If so, does it need to be anchored at the beginning, making it:

```suggestion
        regex = r'^\d+:'
```

Without that, seems it could potentially match anywhere in the name which is brittle.

Reply:
Yes, you're right, it's about stripping off the epoch.